nucliadb_models/src/nucliadb_models/search.py

```diff
@@ -1406,6 +1452,10 @@ class SyncAskResponse(BaseModel):
         title="Answer",
         description="The generative answer to the query",
     )
+    object: Optional[Dict[str, Any]] = Field(
```

Comment:
What about `answer_json` as the field name? To be similar to the `answer_json_schema` input param